src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/public_ip/_update.py

```diff
@@ -91,8 +91,8 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="Idle timeout in minutes.",
             nullable=True,
         )
-        _args_schema.ip_tags = AAZListArg(
-            options=["--ip-tags"],
+        _args_schema.ip_tags_list = AAZListArg(
+            options=["--ip-tags-list"],
             help="Space-separated list of IP tags in 'TYPE=VAL' format.",
```

Comment:
This description is not correct.
```suggestion
            help="Space-separated list of IP tags in 'TYPE=VAL' format.",
```

Reply:
Not exposed to users, anyway, I'll revert it to its swagger definition.